bigframes/core/indexes/base.py

```diff
@@ -454,24 +475,27 @@ def __len__(self):
         return self.shape[0]
 
 
-# Index that mutates the originating dataframe/series
-class FrameIndex(Index):
-    def __init__(
-        self,
-        series_or_dataframe: typing.Union[
-            bigframes.series.Series, bigframes.dataframe.DataFrame
-        ],
-    ):
-        super().__init__(series_or_dataframe._block)
-        self._whole_frame = series_or_dataframe
+class MultiIndex(Index, vendored_pandas_multindex.MultiIndex):
```

Comment:
As much as possible, I'd like our module hierarch to reflect pandas. In pandas, this is defined in `pandas/core/indexes/multi.py`. Anything under `core` is considered private-ish, but these paths do leak out in some of the pandas docs.

https://github.com/pandas-dev/pandas/blob/5232bee8df3b57766c44b62152aa3fdd24e40ada/pandas/core/indexes/multi.py#L224

Let's make a `bigframes/core/indexes/multi.py` file and put this there.

Reply:
done